app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java

```diff
@@ -1106,13 +1106,13 @@ public void onCurrentSessionChange(GeckoSession aOldSession, GeckoSession aSessi
         } else {
             setPrivateBrowsingEnabled(false);
         }
+        waitForFirstPaint();
```

Comment:
We don't seem to be executing the first draw callback in some situations like:
- Open a new tab
- Select the previous tab